src/testUtils/events/defaultApiRequest.ts

```diff
@@ -0,0 +1,49 @@
+export const defaultApiRequest = {
```

Comment:
You don't need the entire object for the event I think. Just the stuff that's going to be useful will do!

Reply:
Also would this be better as just a standard JSON file? I think the convention is to stick it in an `events` folder at the root. That way you can use it with `sam local invoke`.